tests/fsyacc/unicode/test1-unicode.input4.utf8

```diff
@@ -0,0 +1,9 @@
+
+next line tests one unicode character class
+ÄËÖÏÜâæçñõö + id
+next line tests specific unicode characters
+≠ ≠≠ ≈≈ ≈≈≈
+id + id
+next line tests some more random unicode characters
+ǲМНОПРСТУФХЦẀẁẂќ αβΛΘΩΨΧΣδζȚŶǺ
```

Comment:
I will make it more visible that the whole test is about this `ǲ` that is a single title case letter.

Reply:
Done, changed name.